### ISSUE TITLE
Bulk CSV upload of contributions

### ISSUE THREAD

**Issue by christpet** [dis-moi/backend#310]
No need to build an interface yet - just need to build in the ability to upload a CSV file with the correct information to create tens, hundreds, or even thousands of contributions at one time.

- [ ] Figure out columns needed in the CSV file
- [ ] lay out the approach to take
- [ ] implement!

@julienadamcom feel free to add/edit anything that is missing here

**Comment by lutangar** [comment]
> Figure out columns needed in the CSV file

@julienadamcom I'm open to discuss CSV alternatives since we are on the way of implementing an API anyway.

**Comment by JalilArfaoui** [comment]
#161 will be a useful prerequesite :-)

**Comment by JalilArfaoui** [comment]
@julienadamcom if you need any help on this … I can help too

**Comment by christpet** [comment]
Just got an update from @julienadamcom that this is finished... looking for an update from him later today 🙏

**Comment by julienadamcom** [comment]
> Just got an update from @julienadamcom that this is finished... looking for an update from him later today 🙏

Not really, but our real high priority of the moment is done here (without csv): https://github.com/dis-moi/backend/issues/325

**Comment by lutangar** [comment]
Could you push your WIP somewhere around here please? We could already have a look at it while you're working on #325
